File: src/validate_cpf.py
class CPF:
    def __init__(self, cpf_number):
        self.cpf = cpf_number
        self.cpf_clean = self.__clean_cpf()
        self.first_digit = None
        self.second_digit = None
        self.is_valid = self.__is_valid()

    def __clean_cpf(self):
        # TODO function to clean CPF by removing . and - chars
        return None

    def __is_valid(self):
        return None

if __name__ == "__main__":
    # TODO read txt file with test cases and parse each cpf
    pass

File: src/validate_cpf.kt
class CPF(cpf_number: String){
    val cpf = cpf_number
    val cpf_clean = clean_cpf(cpf)

    private fun clean_cpf(cpf: String): String{
        val re = Regex("[^\\d]")
        return re.replace(cpf, "")
    }
}

fun main(){
    val c = CPF("999.999.999-99")
    println(c.cpf)
    println(c.cpf_clean)
}